doc/lighting-setpsbtversion.7.md

```diff
@@ -0,0 +1,65 @@
+lightning-setpsbtversion -- Command for setting PSBT version
+============================================================
+
+SYNOPSIS
+--------
+
+**setpsbtversion** *psbt* *version*
+
+DESCRIPTION
+-----------
+
+The **setpsbtversion** RPC command sets the PSBT version, and returns the base64 result of the conversion. Returns an error if version is invalid.
+
+Elements-based PSETs are a no-op, and does not return an error on invalid version.
```

Comment:
not convinced this is the right behavior, just describing what is implemented!

Reply:
IMO for PSET setting v2 should be a no-op and setting v0 should be an error. Just returning the same result as wally should result in sane behavior. Its unlikely but someone out there may have attempted to implement the v0 PSET spec.

Reply:
yeah, turns out I really didn't need that additional wrapping; could only find a single call location that needed the check(don't export v0 to end user when they didn't ask for it)